### PR TITLE
Add some basic testing to AMIs

### PR DESCRIPTION
**Pull request by edulop91**

### CODE REVIEW

File: packer/ansible/roles/ssh/tasks/configure.yml [hunk 1]
```diff
@@ -15,35 +15,6 @@
         state: present
         validate: '/usr/sbin/sshd -T -f %s'
 
-- name: ssh interactive-keyboard
```
Comment (edulop91) [inline packer/ansible/roles/ssh/tasks/configure.yml]: i'll worry about this later

File: packer/fixtures/terraform/instance/main.tf [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+resource "aws_security_group" "allow_ssh" {
+  name_prefix = "allow_ssh"
+  description = "Allow ssh inbound traffic"
+  vpc_id      = "${var.vpc_id}"
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```
Comment (ryanking) [inline packer/fixtures/terraform/instance/main.tf]: You can get fancy and set to current IP address

Comment (ryanking) [reply packer/fixtures/terraform/instance/main.tf]: (maybe)

File: packer/packer_integration_test.go [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+	vpcID, subnetID := getEnv(t)
+	packerTemplate := "ubuntu16-base.json"
+	amiID := buildAMI(packerTemplate, t)
+	testSSH(amiID, vpcID, subnetID, t)
+}
+
+// helpers
+// -----------------------------
+func getEnv(t *testing.T) (vpcID, subnetID string) {
+	a := assert.New(t)
```
Comment (ryanking) [inline packer/packer_integration_test.go]: neat

**Comment by edulop91** [comment]
@ryanking yeah, i was thinking we generate the AMI as a test candidate and only once all of our tests pass do we go ahead and copy it as as something we want to release

**Pull request merged**